BinaryObjectScanner/Protection/CrypKey.cs

```diff
@@ -17,16 +23,41 @@ public class CrypKey : IPortableExecutableCheck
             if (sections == null)
                 return null;
 
+            // Get the .data/DATA section strings, if they exist
+            var strs = pex.GetFirstSectionStrings("CODE");
```

Comment:
Make sure it's not both `CODE` and `code`

Reply:
Seemingly no ``code`` section in my samples.

Reply:
Commonly seen both. Example: https://github.com/SabreTools/BinaryObjectScanner/blob/e76ce64568a5eaa78334e39519004db869eb832c/BinaryObjectScanner/Protection/CDSHiELDSE.cs#L28